### PR TITLE
Add YAMBURGER

### PR DESCRIPTION
**Pull request by jnewland**
This PR adds a listing for YAMBURGER, an app that validates the syntax of YAML files changed on GitHub Pull requests.

##### Checklist

- [x] If you're adding a listing for your app, be sure your app is in the `/_apps/` folder before opening this Pull Request.
- [x] All comments in frontmatter need to be removed.
- [x] Performs a useful action through the GitHub API that solves an existing problem for developers: continuous integration for YAML
- [x] Is original: for example, it does something not already done by an existing Probot app
- [x] [Has tests](https://github.com/urcomputeringpal/yamburger/tree/master/test)
- [x] [Has documentation](https://github.com/urcomputeringpal/yamburger)
- [x] [Is open source](https://github.com/urcomputeringpal/yamburger)
- [x] [Has a license](https://github.com/urcomputeringpal/yamburger/blob/master/LICENSE)
- [x] [Has a code of conduct](https://github.com/urcomputeringpal/yamburger/blob/master/CODE_OF_CONDUCT.md)
- [x] Has someone willing to at least minimally maintain them for the near future: @jnewland

### CODE REVIEW

**Comment by hiimbex** [comment]
I opened https://github.com/probot/probot.github.io/pull/248 in connection to this build failure.

File: _apps/yamburger.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: YAMBURGER
+description: YAMBURGER finds YAML syntax errors (or YAMBURGERS) on files changed in GitHub Pull Requests.
```
Comment (hiimbex) [inline _apps/yamburger.md]: > YAMBURGER finds YAML syntax errors (or YAMBURGERS) on files changed in GitHub Pull Requests.

Could this become:

> Finds YAML syntax errors (or YAMBURGERS) on files changed in GitHub Pull Requests.

The way the website displays the app names above the description would make this look rather repetitive.

Comment (jnewland) [reply _apps/yamburger.md]: > The way the website displays the app names above the description would make this look rather repetitive.

Great suggestion @hiimbex! I've applied it in 631c038.

**Comment by bkeepers** [comment]
🤘 shipped! https://probot.github.io/apps/yamburger/

**Pull request merged**